src/WikiService.Data/Interfaces/IRubricRepository.cs

```diff
@@ -1,9 +1,19 @@
 ﻿using LT.DigitalOffice.Kernel.Attributes;
+using LT.DigitalOffice.WikiService.Models.Db;
+using System;
+using System.Threading.Tasks;
 
 namespace LT.DigitalOffice.WikiService.Data.Interfaces
 {
   [AutoInject]
   public interface IRubricRepository
   {
+    Task<Guid?> CreateAsync(DbRubric dbRubric);
+
+    Task<bool> DoesExistAsync(Guid rubricParentId);
```

Comment:
the metod check if rubric by id exist, rename rubricParentId to rubricId or rename method